modules/programs/zsh.nix

```diff
@@ -397,6 +397,22 @@ in
             Custom styles for autosuggestion highlighting. See
             {manpage}`zshzle(1)` for syntax.
           '';
```

Comment:
the previous option is not closed, add a `};`

Reply:
Done. Hopefully eval succeeds now.

Reply:
Nope. It does not.